### PR TITLE
remove custom less-middleware with rtltr hack

### PR DESCRIPTION
**Pull request by Pomax**
fixes https://github.com/mozilla/login.webmaker.org/issues/372

### CODE REVIEW

**Comment by Pomax** [comment]
@cadecairos, @alicoding  r?

File: app/http/server.js [hunk 1]
```diff
@@ -108,7 +107,19 @@ module.exports = function (env) {
...
     var optimize = env.get("NODE_ENV") !== "development",
       tmpDir = path.join(require("os").tmpDir(), "mozilla.login.webmaker.org.build");
-    http.use(lessMiddleWare(rtltrForLess({
+
+    // convert requests for ltr- or rtl-specific CSS back to the real filename,
+    // as the rtltr-for-less package was a hack that was never meant to hit production.
+    http.use(function rtltrRedirect(req, res, next) {
+      var path = req.path;
+      if (path.match(/css\/\w+\.(ltr|rtl)\.css/)) {
+        res.redirect(path.replace(/\.(ltr|rtl)/, ""));
```
Comment (alicoding) [inline app/http/server.js]: Are we doing this check in case our CDN still caching those files?

Comment (Pomax) [inline app/http/server.js]: yes. better safe than 404'd

**Comment by alicoding** [comment]
R+

**Pull request merged**